crates/red_knot_python_semantic/resources/mdtest/type_properties/is_gradual_equivalent_to.md

```diff
@@ -157,4 +157,6 @@ def f6(a, /): ...
 static_assert(not is_gradual_equivalent_to(CallableTypeOf[f1], CallableTypeOf[f6]))
 ```
 
+TODO: Overloads
```

Comment:
Unfortunately bidirectional assignability does not imply gradual equivalence, it's more like a static equivalence check but where `Any/Unknown` matches `Any/Unknown`. (Could still potentially be implemented via shared subtyping/assignability logic, with a third strategy for type comparison passed in as a parameter.)